examples/GANwithSelf-taughtLearning/GAN/diffaug.py

```diff
@@ -0,0 +1,76 @@
+# Differentiable Augmentation for Data-Efficient GAN Training
+# Shengyu Zhao, Zhijian Liu, Ji Lin, Jun-Yan Zhu, and Song Han
+# https://arxiv.org/pdf/2006.10738
```

Comment:
If this is an copy of thirdparty source code, please add the original link of repository and make sure the open source license is permit to use this code copy.